gsmphys/radiation_gases.f

```diff
@@ -789,7 +793,7 @@ subroutine gas_update                                             &
           rate = 0.0
         endif
 
-        co2_glb = (co2g1 + rate) * 1.0e-6
+        co2_glb = (co2g1 * co2_scaling + rate) * 1.0e-6
         if ( me == 0 ) then
           print *,'   Global annual mean CO2 data for year',            &
      &              iyear, co2_glb
```

Comment:
Do you want to add a note that you are rescaling the data in that file by co2_scaling?